fvm/src/syscalls/gas.rs

```diff
@@ -4,10 +4,11 @@ use std::str;
 
 use super::Context;
 use crate::gas::Gas;
-use crate::kernel::{ClassifyResult, GasOps, Result};
+use crate::kernel::{ClassifyResult, Result};
+use crate::Kernel;
 
 pub fn charge_gas(
```

Comment:
This syscall will go away once we get rid of the original prove-commit method: https://github.com/filecoin-project/FIPs/pull/820.